radio/src/translations/de.h.txt

```diff
@@ -1120,7 +1120,7 @@
 #define TR_LSW_DESCRIPTIONS            { "Vergleich oder Funktion", "Erste Variable", "Zweite Variable/Konstante", "Zweite Variable/Konstante", "Weitere UND Bedingung für Freigabe des Log Schalters", "ON-Zeit des Log Schalters wenn Bedingung ok", "Mindestdauer der Bedingung damit Log Schalter ON geht" }
 
 //Taranis About screen
-#define TR_ABOUTUS                     "Über OpenTx"
+#define TR_ABOUTUS                     TR(" Info ", "Info")
```

Comment:
I think this should be `TR("Über", "Über EdgeTx")` Since it's translation for "About", and the long form for wider displays is now missing?

Reply:
As you can see in the before screenshot above, the "Ü" was not displayed, so I chose "Info".
Otherwise I took it over as it was done in most other translations.

Reply:
Ah, ok, I thought it might have been offscreen. In that case, lets just implement it like this, and re-add the correct text once we have unicode working again.